src/pages/awsDashboard/awsDashboardWidget.tsx

```diff
@@ -182,8 +182,9 @@ class AwsDashboardWidgetBase extends React.Component<AwsDashboardWidgetProps> {
           title={trendTitle}
           currentData={currentData}
           formatDatumValue={formatValue}
-          formatDatumOptions={{ ...trend.formatOptions, translateFunction: t }}
+          formatDatumOptions={trend.formatOptions}
           previousData={previousData}
+          translateFunction={t}
```

Comment:
I had the first solution working in my code, while the second was more sudo code. Although they both translated without any errors. We've currently have hot loader issues (not related) since we integrated with Insights, so it's possible you encountered that?

If using `i18next.t` inside `formatTooltipLabel` isn't an easy fix, we can merged the code as is. Thanks!